Project.toml

```diff
@@ -3,13 +3,15 @@ uuid = "7a57a42e-76ec-4ea3-a279-07e840d6d9cf"
 keywords = ["probablistic programming"]
 license = "MIT"
 desc = "Common interfaces for probabilistic programming"
-version = "0.3.1"
+version = "0.4"
```

Comment:
In principle the PR doesn't have to be breaking, you could just deprecate `logdensity`. But I guess DynamicPPL is the only/main dependency of AbstractPPL currently, so it doesn't really matter.

Reply:
As we are still in beta, and want to promote DensityInterface adoption, I prefer breaking.  We could leave `logdensity` deprecated in DPPL, though, I don't know how long it has been around before.

Reply:
Sounds good!